management-src/page.js

```diff
@@ -52,10 +52,22 @@ class Page {
       browser.runtime.sendMessage({ type: "setDebuggingEnabled", value: debuggingEnabled.checked });
     }
 
-    const migrationCompleted = document.getElementById("migrationCompleted");
-    migrationCompleted.checked = config.migrationCompleted || false;
-    migrationCompleted.onchange = _ => {
-      browser.runtime.sendMessage({ type: "setMigrationCompleted", value: migrationCompleted.checked });
+    const reminderEnabled = document.getElementById("reminderEnabled");
+    reminderEnabled.checked = config.reminder;
+    reminderEnabled.onchange = _ => {
+      browser.runtime.sendMessage({ type: "setReminder", value: reminderEnabled.checked });
+    }
```

Comment:
Nit: Semi colon.
Does all of this pass linting?

Reply:
Yes it does. Note that we have the same code in many other places.